### ISSUE TITLE
The meat tag could be replaced with FoodComponent#isMeat()

### ISSUE THREAD

**Issue by MoriyaShiine** [apace100/origins-fabric#16]

**Comment by apace100** [comment]
Good point, I didn't even know that was a property! Thanks for the pointer. :)

**Comment by apace100** [comment]
I've added it alongside the meat tag - `isMeat` is used by Minecraft only determine whether a wolf will eat the food. That's why I think it's good to have a tag, to include things such as spider eyes and fish, which are not edible by wolves.

**Comment by florensie** [comment]
What's the status on this? Foods with `isMeat` don't appear to be working.

**Comment by apace100** [comment]
This seems to have gotten lost when I switched to the data-driven power system. Will re-add this in the next update, sorry for the inconvenience.
